torch/csrc/api/include/torch/nn/modules/container/parameterlist.h

```diff
@@ -57,7 +57,7 @@ class ParameterListImpl : public Cloneable<ParameterListImpl> {
   void append(const torch::Tensor& param) {
     bool requires_grad = param.requires_grad();
     register_parameter(
-        c10::to_string(parameters_.size()), std::move(param), requires_grad);
```

Comment:
As `param` is a const reference to `torch::Tensor`, `std::move` can not move it, so that copy happens anyway.